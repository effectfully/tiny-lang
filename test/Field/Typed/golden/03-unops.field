let ?not  = not  T;
let ?neq0 = neq0 0;

let neg'  = neg 0;
let inv'  = inv 0;

let #unpack = unpack 1;