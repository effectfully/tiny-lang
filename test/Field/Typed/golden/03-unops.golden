let ?not'_0 = not T;
let ?neq0'_1 = neq0 0;
let $neg'_2 = neg 0;
let $inv'_3 = inv 0;
let @unpack'_4 = unpack 1;
